app/views/dashboard/_sync_widget.html.erb

```diff
@@ -0,0 +1,30 @@
+<h4 class="header">
+  <%= _("Sync Overview") %>
+</h4>
+
+<% organizations = Organization.current.present? ? [Organization.current] : User.current.allowed_organizations %>
+<% products = Katello::Product.includes(:provider).in_orgs(organizations).syncable_content.syncable %>
+<% products = products.sort { |a,b| b.sync_status[:start_time] <=> a.sync_status[:start_time] } %>
```

Comment:
Still run into an issue here because I have repos that were never synced.  What do you think about filtering those out entirely?

```
comparison of Katello::Product with Katello::Product failed

/home/vagrant/katello/app/views/dashboard/_sync_widget.html.erb:7:in `_a72dd588b6a13f604bc74e92e681a1b7'
app/helpers/dashboard_helper.rb:38:in `render_widget'
app/views/dashboard/index.html.erb:20:in `block (2 levels) in _e2719788fb2b0c6b1fdd4a0c88b2cded'
app/views/dashboard/index.html.erb:14:in `block in _e2719788fb2b0c6b1fdd4a0c88b2cded'
app/views/dashboard/index.html.erb:13:in `_e2719788fb2b0c6b1fdd4a0c88b2cded'
app/controllers/dashboard_controller.rb:8:in `index'
app/controllers/concerns/application_shared.rb:13:in `set_timezone'
app/models/concerns/foreman/thread_session.rb:32:in `clear_thread'
lib/middleware/catch_json_parse_errors.rb:9:in `call'
```